practice/00-HTML-CSS-basics/HelloWorld.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+    <head>
+		<title>My TODO list</title>
+	</head>
+	<body>
+		<h1> My TODO list</h1>
+		<ul>
+			<li>Monday, Wednesday, Friday</li>
+			<br></br>
```

Comment:
Please , you should remove all <br /> within the listings . If you want to create space, you should do it with css. Thank You